### PR TITLE
Add release notes for 0.12.2

### LINKED ISSUE #776: Add release notes for 0.12.2
See https://github.com/submariner-io/releases/issues/363

This should show the PRs merged to release-0.12 since 0.12.1: https://github.com/pulls?q=is%3Apr+user%3Asubmariner-io+label%3Abackport-handled+created%3A%3E2022-04-24+

### PR DESCRIPTION
**Pull request by maayanf24**
Signed-off-by: Maayan Friedman <maafried@redhat.com>

### CODE REVIEW

**Review by nyechiel — COMMENTED** [review]
Thanks for putting this together, @maayanf24. The content LGTM. I will let @dfarrell07 and @tpantelis to fix style and grammar.

File: src/content/community/releases/_index.en.md [hunk 1]
```diff
@@ -4,6 +4,23 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.2
+
+This is a bugfix release:
+
+* Globalnet metrics port will now be opened by default when Globalnet is deployed.
```
Comment (nyechiel) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Globalnet metrics port will now be opened by default when Globalnet is deployed using `subctl`.
```

File: src/content/community/releases/_index.en.md [hunk 2]
```diff
@@ -4,6 +4,23 @@ title = "Releases"
...
+* Submariner ServiceExport now has unique condition types to wait for readiness.
+* The `subctl diagnose` command now supports NAT-discovery port validation.
+* The `cloud prepare` command will now work properly for nodes to which security groups were added manually.
```
Comment (nyechiel) [inline src/content/community/releases/_index.en.md]:
```suggestion
* The `subctl cloud prepare` command will now work properly for nodes to which security groups were added manually.
```

Comment (yboaron) [reply src/content/community/releases/_index.en.md]: I think it should be 'subctl cloud prepare'

Comment (nyechiel) [reply src/content/community/releases/_index.en.md]: Oops, sorry!

File: src/content/community/releases/_index.en.md [hunk 3]
```diff
@@ -4,6 +4,23 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.2
+
+This is a bugfix release:
+
+* Globalnet metrics port will now be opened by default when Globalnet is deployed using `subctl`.
```
Comment (skitt) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Globalnet metrics ports will now be opened by default when Globalnet is deployed using `subctl`.
```

if there are several, or

```suggestion
* The Globalnet metrics port will now be opened by default when Globalnet is deployed using `subctl`.
```

if there is just one.

File: src/content/community/releases/_index.en.md [hunk 4]
```diff
@@ -4,6 +4,23 @@ title = "Releases"
...
+* Submariner ServiceExport now has unique condition types to wait for readiness.
```
Comment (skitt) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Submariner ServiceExport now has unique condition types to simplify waiting for readiness.
```

**Review by skitt — CHANGES_REQUESTED** [review]

File: src/content/community/releases/_index.en.md (hunk 3 above)
Comment (sridhargaddam) [inline src/content/community/releases/_index.en.md]: The current sentence talks about deployment and the first thought i get in my mind is its "subctl join ...". But we are talking about "subctl cloud prepare ...".

So, please clarify that metrics port is opened by default when cloud prepare is done via "subctl"

File: src/content/community/releases/_index.en.md [hunk 5]
```diff
@@ -4,6 +4,23 @@ title = "Releases"
...
+* The `subctl diagnose` command now supports NAT-discovery port validation.
+* The `subctl cloud prepare` command will now work properly for nodes to which security groups were added manually.
```
Comment (sridhargaddam) [inline src/content/community/releases/_index.en.md]: Since cloud prepare is very specific to the infrastructure on which its executed. Can you please clarify to which cloud (AWS/GCP/Azure/RHOS etc) this applies?

**Review by sridhargaddam — CHANGES_REQUESTED** [review]

**Review by dfarrell07 — APPROVED** [review]

File: src/content/community/releases/_index.en.md [hunk 6]
```diff
@@ -4,6 +4,23 @@ title = "Releases"
...
+## v0.12.2
+
+This is a bugfix release:
+
+* The Globalnet metrics port will now be opened by default when Globalnet is deployed using `subctl cloud prepare`.
+* Submariner ServiceExport now has unique condition types to simplify waiting for readiness.
+* The `subctl diagnose` command now supports NAT-discovery port validation.
+* The `subctl cloud prepare rhos` command will now work properly for nodes to which security groups were added manually.
+* The `submariner-operator` namespace is labeled in accordance with _KEP-2579: Pod Security Admission Control_ (default in Kubernetes 1.24)
+to allow the privileged Pods required for Submariner's interactions with iptables.
```
Comment (sridhargaddam) [inline src/content/community/releases/_index.en.md]:
```suggestion
to allow the privileged Submariner Pods.
```

IPtables interaction is just one of the things we do on the nodes. Apart from iptables, we also configure IPsec tunnels, routes, procs entries etc. So lets avoid mentioning iptables and just say that its for privileged submariner pods.

File: src/content/community/releases/_index.en.md [hunk 7]
```diff
@@ -4,6 +4,23 @@ title = "Releases"
...
+* The default namespace for the `subctl diagnose` command is changed to `submariner-operator`.
+* Images are based on Fedora 36.
```
Comment (sridhargaddam) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Submariner pod images are now based on Fedora 36.
```

File: src/content/community/releases/_index.en.md [hunk 8]
```diff
@@ -4,6 +4,23 @@ title = "Releases"
...
+* Fix issues related to Globalnet pods not starting with OpenShift 4.11.
```
Comment (sridhargaddam) [inline src/content/community/releases/_index.en.md]: Can you please share the link of the issue/PR related to this note?

**Review by sridhargaddam — CHANGES_REQUESTED** [review]
Thanks for the PR @maayanf24

File: src/content/community/releases/_index.en.md (hunk 8 above)
Comment (maayanf24) [reply src/content/community/releases/_index.en.md]: issue: https://github.com/submariner-io/submariner/issues/1883
PR: https://github.com/submariner-io/submariner/pull/1895

Comment (sridhargaddam) [reply src/content/community/releases/_index.en.md]: Thanks. The issue was present in both Globalnet pods and route-agent pods and AFAICT is not specific to OCP. So can you please rephrase the sentence as follows.

"Fix issues related to Globalnet and Route-agent pods due to missing grep in the container image."

File: src/content/community/releases/_index.en.md (hunk 7 above)
Comment (Jaanki) [reply src/content/community/releases/_index.en.md]: Agree with Sridhar here,

File: src/content/community/releases/_index.en.md (hunk 6 above)
Comment (tpantelis) [reply src/content/community/releases/_index.en.md]: How about this wording?

```suggestion
to allow the Pods to be privileged.
```

Comment (dfarrell07) [reply src/content/community/releases/_index.en.md]: I think this feedback was missed in the last revision.

File: src/content/community/releases/_index.en.md (hunk 7 above)
Comment (dfarrell07) [reply src/content/community/releases/_index.en.md]: I think this feedback was missed in the last revision.

File: src/content/community/releases/_index.en.md (hunk 8 above)
Comment (dfarrell07) [reply src/content/community/releases/_index.en.md]: I think this feedback was missed in the last revision.

**Review by sridhargaddam — APPROVED** [review]

**Comment by maayanf24** [comment]
@skitt - Can you re-review?

**Review by skitt — APPROVED** [review]
